bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/ExpireManager.java

```diff
@@ -325,6 +329,15 @@ public ExpireConfig(Item item, String configString) throws IllegalArgumentExcept
                     ? configString.substring(commaPos + 1).trim()
                     : null;
 
+            Object ignoreStateUpdatesConfigObject = configuration.get("ignoreStateUpdates");
```

Comment:
Did you remember the `ConfigParser`? Will be a one liner if you use it.

And please refactor the String `"ignoreStateUpdates"` to be a static final constant.

Reply:
Yes, but it's not available here. It's part of `org.openhab.core.config.core` which is not a dependency of `org.openhab.core`.

Reply:
Oh, I see. I missed that